db/migrate/20162812090707_add_events_limit.rb

```diff
@@ -1,5 +1,5 @@
 class AddEventsLimit < ActiveRecord::Migration[5.0]
   def change
-    add_column :events, :limitation, :integer, default: Event::DEFAULT_LIMIT
+    add_column :events, :limitation, :integer, default: 50
```

Comment:
There is not possible to get settings `Ez::Settings[:app, :events, :default_limit]` here?